src/main/sensors/gyro.c

```diff
@@ -170,6 +170,7 @@ PG_RESET_TEMPLATE(gyroConfig_t, gyroConfig,
     .gyro_soft_notch_hz_2 = 200,
     .gyro_soft_notch_cutoff_2 = 100,
     .checkOverflow = GYRO_OVERFLOW_CHECK_ALL_AXES,
+    .gyro_soft_lpf_hz_2 = 200,
```

Comment:
@mikeller should I leave this defaulted to 0 (disabled)?

Reply:
Good point, I had not considered that. But since it's not accessible in configurator for now, and will probably give everybody trying to tune the filters in configurator a hard time if enabled, I think it should be off.

Reply:
@mikeller righto handled in 4dd65a2